awx/api/conf.py

```diff
@@ -71,8 +77,7 @@
     required=False,
     default='',
     label=_('Login redirect override URL'),
-    help_text=_('URL to which unauthorized users will be redirected to log in. '
-                'If blank, users will be sent to the Tower login page.'),
+    help_text=_('URL to which unauthorized users will be redirected to log in. ' 'If blank, users will be sent to the Tower login page.'),
```

Comment:
a bit of weirdness, it's not smart enough apparently to glue the substrings back together.